### PR TITLE
Remove all occurrences of GA code

### PR DESCRIPTION
**Pull request by TamimiGitHub**
This PR removes all occurrences of the the GA code as per #115 

Note: I made sure the last modified metadata is unchanged

### CODE REVIEW

File: markdown/boomi-getting-started/boomi-getting-started.md [hunk 1]
```diff
@@ -5,21 +5,21 @@ tags:
 categories: Boomi
 environments: Web
 status: Draft
-feedback link: 
-analytics account: UA-3921398-10
+feedback link:
```
Comment (Mrc0113) [inline markdown/boomi-getting-started/boomi-getting-started.md:8]: point to community?

File: markdown/azure2Solace/azure2Solace.md [hunk 2]
```diff
@@ -2,18 +2,17 @@ author: Sumeet Koshal
...
 id: azure2Solace
 tags: iguide
-categories: Azure, C# 
+categories: Azure, C#
 environments: Web
 status: published
 feedback link: https://github.com/solacese/azure2Solace
-analytics account: UA-3921398-10
-
+analytics account:
```
Comment (Mrc0113) [inline markdown/azure2Solace/azure2Solace.md:9]: remove empty entry

File: markdown/azure2Solace/azure2Solace.md [hunk 3]
```diff
@@ -24,70 +23,73 @@ You will find the Azure function code [here](https://github.com/solacese/azure2S
 To learn how to stream data from Solace PubSub+ broker to Azure messaging services, click [here](https://docs.solace.com/Configuring-and-Managing/HTTP-Header-Authentication-Azure-Event-Hub.htm).
 
 ## High Level Architecture of this codelab's goal
+
```
[the remark this answers is not in the document]
Comment (TamimiGitHub) [reply markdown/azure2Solace/azure2Solace.md:26]: Checked and extra line does not affect the rendered duration time

File: markdown/design-document-discover-develop-event-driven-apis/export.sh [hunk 4]
```diff
@@ -9,4 +9,4 @@ codelab_markdown_filename=`ls *.md`
 codelab_dir=`basename $codelab_markdown_filename .md`
 rm -fr $codelab_dir
 
-claat export -ga UA-3921398-10 -o ../../codelabs/ $codelab_markdown_filename
+claat export -o../../codelabs/ $codelab_markdown_filename
```
[the remark this answers is not in the document]
Comment (TamimiGitHub) [reply markdown/design-document-discover-develop-event-driven-apis/export.sh:12]: Fixed 👍

File: markdown/boomi-getting-started/boomi-getting-started.md (hunk 1 above)
Comment (TamimiGitHub) [reply markdown/boomi-getting-started/boomi-getting-started.md:8]: Good call - fixed 👍

File: markdown/azure2Solace/azure2Solace.md (hunk 2 above)
Comment (TamimiGitHub) [reply markdown/azure2Solace/azure2Solace.md:9]: Done 👍

**Comment by TamimiGitHub** [comment]
Fixed in #117

**Pull request closed without merging**